activiti-cloud-query-service/activiti-cloud-services-query/activiti-cloud-services-query-repo/src/test/java/org/activiti/cloud/services/query/app/repository/utils/ProcessInstanceHelper.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2017-2020 Alfresco Software, Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.activiti.cloud.services.query.app.repository.utils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.activiti.cloud.services.query.model.ProcessInstanceEntity;
+
+public class ProcessInstanceHelper {
```

Comment:
Please convert these method to static, no need an instance for this utility class